db/users.js

```diff
@@ -0,0 +1,144 @@
+const client = require("./client");
+const bcrypt = require('bcrypt');
+const SALT_COUNT = 10;
+
+async function createUser({
+  firstName,
+  lastName,
+  password,
+  phone,
+  email,
+  isActive,
+  isAdmin
+}) {
+  try {
+    const hashedPassword = await bcrypt.hash(password, SALT_COUNT);
+
+    const { rows: [ user ] } = await client.query(`
+      INSERT INTO users(firstName, lastName, password, phone, email, isActive, isAdmin)
+      VALUES ($1, $2, $3, $4, $5, $6, $7)
+      ON CONFLICT (email) DO NOTHING
+      RETURNING *;
+    `, [firstName, lastName, hashedPassword, phone, email, isActive, isAdmin]);
+
+    delete user.password;
+
+    return user;
+  } catch (error) {
+    console.error(error)
+  }
+}
+
+async function getAllUsers() {
+  try{
+      const { rows } = await client.query(`
+          SELECT *
+          FROM users
+      `);
+  
+      return rows;
+  } catch (error) {
+      console.error(error)
+  }
+}
+
+async function getUser({ 
```

Comment:
getUser should only need email and password as parameters.

Reply:
Dont